on_push.js

```diff
@@ -17,14 +17,9 @@ exec('npm install', {async:true}, function () {
 // Publish viewer to gh-pages
 //
 exec('gulp web', {async:true}, function() {
-//
-// Sign the extension
-//
-exec('node ' + path.join(__dirname, 'signxpi.js') + ' ./build/gh-pages/extensions/firefox/pdf.js.xpi', {async:true}, function () {
```

Comment:
Doesn't https://github.com/mozilla/botio-files-pdfjs/pull/21/files#diff-4e0084538624ae99e2e76e87012cf654L43 also need to be removed?